lib/crawly/utils.ex

```diff
@@ -139,6 +139,31 @@ defmodule Crawly.Utils do
     end
   end
 
+
+  @doc """
+  Returns a list of known modules which implements Crawly.Spider behaviour
+  """
+  @spec list_spiders() :: [module()]
+  def list_spiders() do
+    Enum.reduce(
+      get_modules_from_applications(),
+      [],
+      fn mod, acc ->
+        try do
+          beh = Keyword.get(mod.__info__(:attributes), :behaviour)
+
+          case beh == [Crawly.Spider] do
```

Comment:
should use Enum.any?/2 check instead, in case a module implements multiple (custom) behaviours

Reply:
Nice catch. I have added a fix & test for it.